src/main/java/ch/njol/skript/expressions/ExprArrowsStuck.java

```diff
@@ -0,0 +1,92 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.expressions;
+
+import org.bukkit.entity.LivingEntity;
+import org.bukkit.event.Event;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.classes.Changer.ChangeMode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.expressions.base.SimplePropertyExpression;
+import ch.njol.util.coll.CollectionUtils;
+
+@Name("Arrows Stuck")
+@Description("The number of arrows stuck in a living entity.")
+@Examples("set player's stuck arrows to 5")
+@Since("INSERT VERSION")
+public class ExprArrowsStuck extends SimplePropertyExpression<LivingEntity, Number> {
+
+    static {
+    	if (Skript.methodExists(LivingEntity.class, "getArrowsStuck")) {
+    		register(ExprArrowsStuck.class, Number.class, "stuck arrow[s]", "livingentities");
```

Comment:
wouldn't it be nice to have `add 1 to arrows stuck in player`? this syntax doesn't allow that

Reply:
in fact i'm not sure if `arrows stuck of player` makes sense at all

Reply:
Hmm... that's a good point. Will probably not be able to use a SPE for this expression.

Reply:
you can use an SPE (and should), you just won't be able the prebuilt registration method

Reply:
Hopefully the new pattern is better.